src/components/typography/placeholder/placeholder.tsx

```diff
@@ -0,0 +1,15 @@
+import clsx from 'clsx';
+import { PlaceholderProps } from './types';
+import { FC } from 'react';
+
+export const Placeholder: FC<PlaceholderProps> = ({ text, size, className = 'text-inherit' }) => {
```

Comment:
If you want you can set size default to PlaceholderSize.M and make it optional, since M is the most (or even only) used size